src/main/resources/assets/powersuits/lang/zh_CN.lang

```diff
@@ -1,4 +1,3 @@
-#zh_CN
 Made by users: crafteverywhere, 3TUSK
```

Comment:
Actually... I suggest add your name into it.
--->`Made by users: crafteverywhere, 3TUSK, sf_chipan`